genume/console.py

```diff
@@ -0,0 +1,9 @@
+from genume.logging.logger import init as initLogger
```

Comment:
Duplicate of __main__.
Just move console_entry into __main__.
No need for duplicating startup code.